e2e/vm/host_gateway_ip_test.go

```diff
@@ -0,0 +1,57 @@
+package vm
+
+import (
+	"os/exec"
+	"path/filepath"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"github.com/onsi/gomega/gexec"
+	"github.com/runfinch/common-tests/command"
+	"github.com/runfinch/common-tests/option"
+
+	"github.com/runfinch/finch/e2e"
+)
+
+const defaultImage    = "public.ecr.aws/docker/library/alpine:latest"
+
+var testHostGatewayIp = func(o *option.Option, installed bool) {
```

Comment:
Could you do some Finch config file content check for each case? Similar to here https://github.com/runfinch/finch/blob/baf645f250344a42bc4ee3ea53e97b2c82dbb55a/e2e/vm/config_test.go#L116.

Ensure there is no host-gateway-ip if it is not specified and ensure it exists if specfied.

Reply:
@ningziwen  how do I access the toml file for nerdctl?

Reply:
I think it is  similar to [here](https://github.com/runfinch/finch/blob/baf645f250344a42bc4ee3ea53e97b2c82dbb55a/e2e/vm/config_test.go#L95C19-L95C46).
```
finchCfgBuf = os.ReadFile(filepath.Clean(finchconfigFilepPath))
err = yaml.Unmarshal(finchCfgBuf, &finchCfg)
```

Reply:
@ningziwen I thought this is for the lima yaml file, the host_gateway_ip will be update to the nerdctl toml file, so how can I get that file to check it?

Reply:
Yeah but you can replace it to finch config file instead of Lima. You already have `finchConfigFilePath` as a variable and already use it in this file (at line 24).

I think it is not necessary to check nerdctl toml file in Finch e2e tests because finch e2e tests test Finch user visible behaviours. (although it doesn't hurt to have)

Reply:
oh I see what you mean, thank you very much